### PR TITLE
Lesson10 refactor

### PR DESCRIPTION
**Pull request by billmwong**

### CODE REVIEW

File: lessons/10-deployment-scaling/README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+  - [Deployment](#deployment)
+  - [Scaling](#scaling)
+- [Lesson 10 - In-Class Exercises](#lesson-10---in-class-exercises)
+- [Before Class 11 \(Tuesday 2/28/17\)](#before-class-11-tuesday-22817)
+- [Lab 1](#lab-1)
+
+<!-- /MarkdownTOC -->
+
+
 #Lesson 10 - Deployment and Scaling
```
Comment (yunhsincynthiachen) [inline lessons/10-deployment-scaling/README.md:16]: Under the deployment section, they haven't deployed to heroku yet

File: lab1/README.md [hunk 2]
```diff
@@ -21,4 +11,4 @@ You must construct a wikipedia-like clone using your chosen framework. You shoul
 If you want to make the wiki have a special topic area (like harry potter or some game) you are encouraged to do so.
 Brownie points will be awarded for awesome looking websites whose themes match the topic area, though having all the functionallity should come before making it pretty.
 
-##Due: Next Friday, 3/4/16
+##Due: Tuesday, 2/28/17
```
Comment (yunhsincynthiachen) [inline lab1/README.md:14]: Is there a homework submission survey?

**Review by yunhsincynthiachen — COMMENTED** [review]
Other than my two comments, everything looks good!

File: lab1/README.md (hunk 2 above)
Comment (billmwong) [reply lab1/README.md:14]: not yet, I can make one

File: lessons/10-deployment-scaling/README.md (hunk 1 above)
Comment (billmwong) [reply lessons/10-deployment-scaling/README.md:16]: good catch, will fix

**Pull request merged**